README.md

```diff
@@ -22,13 +22,27 @@ This DSL relies on the Ruby [pact-mock_service][pact-mock-service] gem to provid
 ## Installation
 Note: see [Upgrading][upgrading] for notes on upgrading from 0.2 to 0.3
 
-### Install the [pact-mock_service][pact-mock-service]
+### Install the Pact Mock Service
 
-Run `sudo gem install pact-mock_service -v 2.1.0` in your terminal.
+Install the [pact-ruby-standalone][pact-mock-service-standalone] as per installation instructions.
 
+#### Xcode Setup
 In Xcode, edit your scheme and add _pre-_ and _post-actions_ for your `Test` step to run the provided scripts in `./scripts/` folder. Make sure you select your target in _Provide build settings from_ the drop down menu.
+
+Example if you installed _pact-ruby-standalone_:
+```
+# Pre-actions
+PATH=/full/path/to/your/standalone/pact/bin:$PATH
+"$SRCROOT"/scripts/start_server.sh
+
+# Post-actions
+PATH=/full/path/to/your/standalone/pact/bin:$PATH
+"$SRCROOT"/scripts/stop_server.sh
+```
+
+of if you installed Pact mock service using `gem install pact-mock_service`:
```

Comment:
Should we just get rid of the details for the non-standalone version? Make things simpler...

Reply:
Sounds good to me :)